about.html

```diff
@@ -0,0 +1,126 @@
+                    </p>
+                </div>
+                <div class="paragraph_two">
+                    <p>Please contact us per Email of any further questions below</p>
+                    <h4>RecordLabel2023@submissions.org</h4>
+                </div>
+            </div>
+        </section>
+        <section class="work_section">
+            <div class="work_section_content">
+                <div class="main_program">
+                    <h2> create Record Label submissions 2023 logo</h2>
+                    <div class="line_program"></div>
+                    <p> 
+                        The logo Record Label submissions 2023 was decided through different season of concerts
+                    </p>
+                </div>
+                <div class="main_program_image">
+                    <img src="images/label.png" alt="">
+                </div>
+            </div>
+
+        </section>
+        <section class="features">
+            <div class="features_container">
+                <div class="featured_artists">
+                    <h2>See The Past Concerts</h2>
+                    <div class="line_program"></div>
+                    <p>
+                        Take a look at The Two previuos concerts that took place in berlin and calfornia
+                    </p>
+                </div>
+                <div class="past_concerts">
+                    <div class="first_past_concert">
+                        <h2>2018</h2>
+                        <p>Monsters of Rock 2018 in berlin</p>
+                    </div>
+                    <div class="second_past_concert">
+                        <h2>2018</h2>
+                        <p>Monsters of Rock 2018 in berlin</p>
+                    </div>
+                </div>
+                                    
+             </div>
+
+            </div>
+        </section>
+
+    </main>
+    <footer>
+         <section class="footer">
+            <div class="footer_container">
+                <div class="footer_header">
+                    <h2>Partners</h2>
+                    <div class="line_partner"></div>
+                </div>
+                <div class="partner_logo">
+                    <div class="partner_image_logo1">
+                        <img src="images/google_.svg" alt="" class="image1">
+                        <img src="images/Twitter  .svg" alt="" class="image2">
+                        <img src="images/Airbnb .svg" alt="" class="image3">
+                        <img src="images/Airbnb .svg" alt="" class="image3">
+                        <img src="images/Airbnb .svg" alt="" class="image3">
+                    </div>
+                </div>
+            </div>
+            <div class="lower_footer">
+                <img src="images/label.png" alt="">
+                <p>@2023 The monsters of rock concerts in gotham </p>
+            </div>
+
+         </section>
+    </footer>
```

Comment:
- [ ] In the desktop version, the footer on the About page has a dark background. The partners' section is not displayed on the About page according to the design. Please implement it accordingly.

**Your design**

![Screenshot from 2023-02-09 16-41-16](https://user-images.githubusercontent.com/17243325/217846486-0d18ba7f-c1ad-4317-9bb1-661610545af5.png)

**Template**

![Screenshot from 2023-02-09 16-40-58](https://user-images.githubusercontent.com/17243325/217846577-cf8aba98-9a28-4f19-856c-b207a7d493f7.png)